From 0e36afd2d09f04ab217d69e89256b6ea79d6d00c Mon Sep 17 00:00:00 2001
From: stack <stack@apache.org>
Date: Thu, 4 Feb 2016 16:23:32 -0800
Subject: [PATCH 063/217]  HBASE-15210 Undo aggressive load balancer logging
 at tens of lines per millisecond; ADDENDUM changing
 LOG.debug to LOG.trace -- noticed by matteo

---
 .../hbase/master/balancer/BaseLoadBalancer.java    |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java b/hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java
index fd1ddd8..3fa14da 100644
--- a/hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java
+++ b/hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/BaseLoadBalancer.java
@@ -827,7 +827,7 @@ public abstract class BaseLoadBalancer implements LoadBalancer {
           }
         }
         if (LOG.isTraceEnabled()) {
-          LOG.debug(" Lowest locality region index is " + lowestLocalityRegionIndex
+          LOG.trace(" Lowest locality region index is " + lowestLocalityRegionIndex
             + " and its region server contains " + regionsPerServer[serverIndex].length
             + " regions");
         }
-- 
1.7.9.5

